### ISSUE TITLE
xfce super key

### ISSUE THREAD

**Issue by ZhengshuaiPENG** [manjaro/manjaro-xfce-settings#3]
hello, first thanks for amazing work of manjaro.
I have a problem with my xfce edition. The super key was binded to whiskermenu, and i use the <super + space> as another shortcut(trigger fcitx inputmethod), but this shortcut doesn't work unless I unbind the super key to whiskermenu. 
It seems that the super key as a shortcut alone will inhibit other shortcuts with superkey.
Any ideas?

Thanks in advance.

**Comment by oberon2007** [comment]
If I understand you right you are trying to use the Super key as a mod key like the Alt keys. My experience with Xfce is that it just doesn't work. I don't know for sure or why exactly, but I had tried several things, too and I didn't succeed. In the end I just used keybindings avoiding Super :wink: 
Maybe that would be a question for the [Xfce forum](https://forum.xfce.org/) In any case I don't think it is manjaro specific or that it related to or caused by our configuration :smile:
